janus/src/hpke.rs

```diff
@@ -169,18 +169,18 @@ pub mod test_util {
         message::{HpkeAeadId, HpkeConfig, HpkeConfigId, HpkeKdfId, HpkeKemId, HpkePublicKey},
     };
     use hpke::{kem::X25519HkdfSha256, Kem, Serializable};
-    use rand::thread_rng;
+    use rand::{thread_rng, Rng};
 
     /// Generate a new HPKE keypair and return it as an HpkeConfig (public portion) and
     /// HpkePrivateKey (private portion).
     pub fn generate_hpke_config_and_private_key() -> (HpkeConfig, HpkePrivateKey) {
         let (private_key, public_key) = X25519HkdfSha256::gen_keypair(&mut thread_rng());
         (
             HpkeConfig::new(
-                HpkeConfigId::from(0),
+                HpkeConfigId::from(thread_rng().gen::<u8>()),
                 HpkeKemId::X25519HkdfSha256,
-                HpkeKdfId::HkdfSha512,
-                HpkeAeadId::ChaCha20Poly1305,
+                HpkeKdfId::HkdfSha256,
+                HpkeAeadId::Aes128Gcm,
```

Comment:
This solves my problem, but you may find it useful to support multiple ciphersuites for your own internal testing purposes.

Reply:
True, but this change doesn't make us any worse off in terms of coverage across HPKE configs, so I would punt on testing the full space of HPKE configs to a future change.